src/utils/oauth2_util.py

```diff
@@ -1,8 +1,57 @@
 from datetime import datetime, timedelta
 
+from src.constants.oauth2 import Roles, Scopes
+from src.constants.http import HttpMethods
+from src.lib.entities.secured_http_request_uri import SecuredHttpRequestUrl
+from src.lib.entities.secured_http_request_Url_permissions import (
+    SecuredHttpRequestUrlPermissions,
+)
 import bcrypt
 import jwt
 
+from src.exceptions.exceptions import UnAuthorizedEndpoint
+
+
+def get_endpoint_roles_scopes_access_permission(secured_http_request_url):
+
+    endpoint_roles_map = {
```

Comment:
Please put the map definition outside of this function

Reply:
done.